src/js/chromecast/ChromecastSessionManager.js

```diff
@@ -208,13 +207,15 @@ class ChromecastSessionManager {
       var player = this.player,
           currentTime = sessionCurrentTime || player.currentTime(),
           wasPlaying = sessionPlaying || !player.paused(),
-          sources = player.currentSources();
+          sources = this._playerSrc || player.lastSource_.player;
```

Comment:
Why not more obvious

```suggestion
          sources = this._playerSrc || player.currentSources();
```

?